lib/openscap_parser/value_definitions.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'openscap_parser/value_definition'
+
+module OpenscapParser
+  # Methods related to parsing values
+  module ValueDefinitions
+    def self.included(base)
+      base.class_eval do
```

Comment:
There's an included callback or something similar that does exactly what you want on compile time. Your code runs at runtime, making it slower.
```suggestion
    included do
       ...
     end
```